common/scripted_effects/00_scripted_effects.txt

```diff
@@ -18846,11 +18829,9 @@ china_war_spawn_troops_effect = {
 					home = event_target:spawn_province
 					troops = {
 						archers = { 107 107 }
-						light_infantry = { 22 22 }
-						heavy_infantry = { 106 106 }
-						pikemen = { 96 96 }
```

Comment:
I would summarize heavy and light infantry... or even give mages or something else.